src/locale/et.js

```diff
@@ -1,6 +1,26 @@
 // Estonian [et]
 import dayjs from 'dayjs'
 
+function relativeTimeWithTense(number, withoutSuffix, key, isFuture) {
+  const format = {
+    s: ['mõne sekundi', 'mõni sekund', 'paar sekundit'],
+    ss: ['%d sekundi', '%d sekundit'],
```

Comment:
Think we don't need this line?

Reply:
Yeah, I wondered why that was there since I don't think moment uses it either.